apps/sms/views/conversation/js/conversation.js

```diff
@@ -642,10 +642,20 @@ var ConversationView = {
       args.draftId ? this.handleDraft(+args.draftId) : Promise.resolve();
 
     return draftPromise.then(() => {
-      if (args.focusComposer) {
-        Compose.focus();
-      } else {
-        this.recipients.focus();
+      switch (args.focus) {
+        case 'composer':
+          Compose.focus();
+          break;
+        case 'recipients':
+          this.recipients.focus();
+          break;
+        default:
+          if (this.hasValidRecipients()) {
```

Comment:
If I understand correctly, `this.hasValidRecipients` only checks that the input value is not empty and finite right? So actually, it actually checks whether or not we have something.In case of only invalid recipients, maybe we'd want to focus the recipients input instead of the composer... Yep, we'll see what happens :-p